### DISCUSSION TITLE
AmigaOS 4.1

### DISCUSSION THREAD

**Discussion by ecstaticax** [utmapp/UTM#3932] [UTM Help (macOS)]
I've found a guide to install AmigaOS through this qemu command line:

` ./qemu-system-ppc -machine sam460ex -rtc base=localtime \
 -drive if=none,id=cd,file=Sam460InstallCD-53.58.iso,format=raw \
 -device ide-cd,drive=cd,bus=ide.1`

* UTM Version: 3.1.4 (52)
* OS Version: MacOS 12.3.1
* Intel or Apple Silicon? Silicon

I tried to create a VM, with PowerPC machine and sam460ex processor, but sadly it never starts. What it the configuration I need to create in UTM to replicate this command line?
Thanks

**Comment by osy** [comment]
I'm not familiar with PPC. If you go into the QEMU menu in the settings you can see all the command line it generates. Then you can play with the settings and see how that changes it.

**Comment by timocock** [comment]
I can confirm this. UTM builds a command line that includes multiple spice elements even if no spice dependencies are enabled in the GUI. This prevents the amiga guest from running because there is no spice client for amiga. You have a dialog that lets you see all the command line settings, but you don't offer any way to edit it.
